functions-iterators/starter.js

```diff
@@ -29,11 +29,26 @@ This will help us track your progress and it will give you practice writing good
 
 // PART 0: Just DO it!!
 
+function justDoIt(){
+  return "JUST DO IT!";
+}
+
+console.log(justDoIt('Norah'));
 
 
 // PART 1: Big or Small String?
 
 
+function oddOrEvenString(word){
```

Comment:
this function will always return `"This word is short"` because you didn't do a legal condition you can't compare strings with number how you gonna solve this `" is 'Norah'  less than 10 "` it doesn't make any sense we need to compare the length of `'Norah'`  which is 5 characters with number 10 so we can compare numbers with numbers. The condition will be something like this ` if (word.length > 10  )`